testplan/testing/multitest/driver/base.py

```diff
@@ -159,6 +142,7 @@ def __init__(
         super(Driver, self).__init__(**options)
         self.extracts = {}
         self._file_log_handler = None
+        self._metadata_extractor = metadata_extractor
```

Comment:
we can refer to this as self.cfg.metadata_extractor, and no need for an additional data field.

Reply:
Sure, let me remove it and the property as well.